src/node/rpc/common_endpoint_registry.h

```diff
@@ -75,6 +75,7 @@ namespace ccf
       };
       make_command_endpoint("tx", HTTP_GET, json_command_adapter(get_tx_status))
         .set_auto_schema<GetTxStatus>()
+        .set_execute_locally(true)
```

Comment:
I don't think this is right, because the session could otherwise be forwarded and the tx status will be out of sync with the results of execution.

Reply:
As discussed offline - I added a new endpoint called local_tx which is the same as tx but sets "set_execute_locally"